docs/reference/cli/apps.md

```diff
@@ -117,6 +117,50 @@ Enable termination protection
     $ convox apps lock
     Locking myapp... OK
 
+## apps params
```

Comment:
Let's just leave app and rack params out of docs until we use them, feel like this will be confusing for not much gain.